cognite_toolkit/cdf_tk/load.py

```diff
@@ -220,8 +220,9 @@ class Loader(ABC, Generic[T_ID, T_Resource, T_ResourceList]):
     list_cls: type[CogniteResourceList]
     dependencies: frozenset[Loader] = frozenset()
 
-    def __init__(self, client: CogniteClient):
+    def __init__(self, client: CogniteClient, ToolGlobals: CDFToolConfig):
         self.client = client
+        self.ToolGlobals = ToolGlobals
```

Comment:
The `ToolGlobals` was passed into `create` and `load_resource` and now I needed it in `delete`, so I think it is better to have it as part of the state of the Loader.